src/private_pgn/pgn_write_impl.ts

```diff
@@ -229,6 +229,9 @@ export function writeGame(game: Game) {
 	result += writeOptionalHeader('Variant', formatVariant(variant));
 	result += writeOptionalIntegerHeader('WhiteElo', game.playerElo('w'));
 	result += writeOptionalHeader('WhiteTitle', game.playerTitle('w'));
+	for (const [tag, value] of Object.entries(game.tags())) {
```

Comment:
Whatever the solution chosen to handle the `Termination` tag, `pgn_read` must be modified as well, for consistency.